docs/add-ons/testing.md

```diff
@@ -9,7 +9,7 @@ The fastest and recommended way to develop add-ons is using a local Visual Studi
 - Copy the [`tasks.json`](https://github.com/home-assistant/devcontainer/raw/main/addons/tasks.json) file to `.vscode/tasks.json` in your repository.
 - Open the root folder inside VS Code, and when prompted re-open the window inside the container (or, from the Command Palette, select 'Rebuild and Reopen in Container').
 - When VS Code has opened your folder in the container (which can take some time for the first run) you'll need to run the task (Terminal -> Run Task) 'Start Home Assistant', which will bootstrap Supervisor and Home Assistant.
-- You'll then be able to access the normal onboarding process via the Home Assistant instance at `http://localhost:8123/`.
+- You'll then be able to access the normal onboarding process via the Home Assistant instance at `http://localhost:7123/`.
```

Comment:
As shown in your screenshot (you've provided), the port is forwarded from internal 7123 to external 8123

![CleanShot 2023-07-30 at 19 31 48](https://github.com/home-assistant/developers.home-assistant/assets/195327/3f761776-b5ab-4371-98e1-1e10024deaf6)

See also here: <https://github.com/home-assistant/devcontainer/blob/4a9ab3fad258c30a2ea330269d7f0f413d3a1a86/addons/devcontainer.json#L4>

The provided change, is therefore, thus incorrect.

../Frenck

Reply:
Hello @frenck,
I didn't write this PR out of nowhere, I made it because I followed the tutorial and I actually have to open `localhost:7123` in my browser in order to see the Home Assistant instance. There is nothing on `localhost:8123`, and I can send you screenshots if you don't believe me !

Also, this is what I found in the [docker documentation](https://docs.docker.com/network/#published-ports):

Flag value | Description
-- | --
-p 8080:80 | Map TCP port 80 in the container to port 8080 on the Docker host.

So if I understand correctly, `"appPort": ["7123:8123", "7357:4357"],` should mean:

> Map TCP port 8123 in the container to port 7123 on the Docker host, and map TCP port 4357in the container to port 7357on the Docker host

Are you certain my change is incorrect?

Reply:
And if I could add something @frenck , when you say

> As shown in your screenshot (you've provided), the port is forwarded from internal 7123 to external 8123

I'm not sure you're interpreting the `docker ps` output correctly. Just in case, here is a screenshot from the [`docker ps` documentation](https://docs.docker.com/engine/reference/commandline/ps/):

![image](https://github.com/home-assistant/developers.home-assistant/assets/17952318/497bd337-3f14-42c9-b20b-06634366378b)

As you can see, the **first port** you see is the port on the host machine, and the **second one** is the container port.

This means I should call port 32738 on my computer to target the port 80 of busybox.

And in my screenshot, it means that I should call port 7123 on my computer to target the port 8123 of home-assistant.

And after all, this makes sense, right? 8123 is the default home assistant port, so why would the home assistant docker container expose the port 7123?